RUNNING_DOCKER.RST

```diff
@@ -0,0 +1,369 @@
+    box and upload a certificate. If you are a member of SFDO please visit the following
+    quip link https://salesforce.quip.com/sm9bA3R6GwiK, and look under section 5 of SFDX
+    Command Line Tool Setup & Install
+
+SFDX_HUB_KEY:          
+    SFDX_HUB_KEY is the private key that was used to create the certificate.
+    Shared through last pass. In the form of a pem key. 
+    Called `SFDX Hub Org Key` n Release Engineering folder.
+    
+FORMATTING SFDX_HUB_KEY in .env
+    IMPORTANT to format on a single line, escaping each newline in the key with ``\n``
+    character otherwise the variable will not be read correctly. Must look like the 
+    following example and CANNOT BE IN QUOTES:
+
+::
+
+    SFDX_HUB_KEY=-----BEGIN RSA PRIVATE KEY-----\nMINDSJIBAAAFDVCXwVAe1YBRi+WpkTp02mOPbgj9NgjpwKQEOJugAqdzBdprBxTs\nMINDSJIBAAAFDVCXwVAe1YBRi+WpkTp02mOPbgj9NgjpwKQEOJugAqdzBdprBxTs\nMINDSJIBAAAFDVCXwVAe1YBRi+WpkTp02mOPbgj9NgjpwKQEOJugAqdzBdprBxTs\nv4fU8l7TeYVQVvSdWJmN3sBZ4bnG3GSu1u6viGQwxulxtJrLnclEgL2Tq0npRn/x\nMINDSJIBAAAFDVCXwVAe1YBRi+WpkTp02mOPbgj9NgjpwKQEOJugAqdzBdprBxTs\nMINDSJIBAAAFDVCXwVAe1YBRi+WpkTp02mOPbgj9NgjpwKQEOJugAqdzBdprBxTs\nMINDSJIBAAAFDVCXwVAe1YBRi+WpkTp02mOPbgj9NgjpwKQEOJugAqdzBdprBxTs\nDMG9uoYPD4X0rkKz/4PI2jcO4NgkWfTiQY0yEDQNM31Sfcw5lNSeKHrrnG7fHx3q\nu9fb7GxWMi74LBlMVlseREzfYRyUI7ukPZNgdvAGbp3TI0ITAQTbTzKPR4FdyZbm\nysuDXZuQpbifXxBKPVVYHxbdEYkabK4FKeB1cNRI72T0jt+r6DqFTjfpJHs/FjEo\nq86HWtHWGh1AYaIi5LBMLQ1tNEcSNvvZW49AsUISqJRFwFvwubBhLh36DaucM4aI\nWPLQUeUCgYEA37+Qy6o3vvfwj0pJ4Ecqo5FRZkxBbUmVTdr1RVPAFxRchsKzsvx4\nWKRDkmIlvf/vpaB4cUsYDZVOd1qGXciFQODk+FfLbOCDbcR1qv87YL/tKNRO/sox\nBt3yS6vyCokn48Ycaqs+tYcHC2O0Vaye/VvwwUSQMLLVdGR84N2hzX8CgYEA3S15\ndqEiWI8a27EX4AD4q9avNJJCwkO5B9/YBnZBpy1DcFSozP5JfgoH1ilK4tmiXjZO\n3Y+oTcKRUKOSQPjv8obTt3N3xtdabWMW6sH31kOfiKOmDg2lw/UjYQ+xO5FBE/Pi\nOR4XRbhSe04dJ+U2Gik38f/WtgA9h53YOeAJ5UMCgYA2kFLRN+tsSK6DYwxtAy3k\nwZVmKwZxjlY4rELP60KW3kJKIsULywHWLAjGc+TcVsOsUlvM1RFCjryZ4puN106X\nMINDSJIBAAAFDVCXwVAe1YBRi+WpkTp02mOPbgj9NgjpwKQEOJugAqdzBdprBxTs\nMINDSJIBAAAFDVCXwVAe1YBRi+WpkTp02mOPbgj9NgjpwKQEOJugAqdzBdprBxTs\nMINDSJIBAAAFDVCXwVAe1YBRi+WpkTp02mOPbgj9NgjpwKQEOJugAqdzBdprBxTs\nDtfenYxFW9Iqj58oCzDuUJGWkA4lolYMkcbvEhE2fhOTNH9UdFyhC6WDQuaFnr1x\nbC4LAoGAbzqfS4vF+kloxneGdWJnAiibvEEUWVmMZ4GMF0a7w0x2l+jwiGT2Kt8P\nC5VdZvMMktzfTHynq6j6BfnSYCBJFNp1EbwZksGtEnT4ggCdIVNY+N1wVeok1vp/\n17/R87a1O62MeA5gBeGdpoMof/XrFVUdb/kSXyNt8miUeLOez/M=\n-----END RSA PRIVATE KEY-----
+
+SFDX_HUB_USERNAME: 
+    This represents the username used to login to your sfdx hub account
+
+CONNECTED_APP_CLIENT_ID:
+    This represents the client id of the packaging org that MetaCI is running builds on
+
+CONNECTED_APP_CLIENT_SECRET: 
+    This represents the secret of the packaging org configured for MetaCI
+
+CONNECTED_APP_CALLBACK_URL:
+    This represents the packaging org's callback url 
+
+To acquire the connected_app variables if you are already properly configured with cci:
+
+::
+    
+    cci service info connected_app
```

Comment:
This won't work for most people -- CumulusCI comes with a default connected app that does not use a client secret.

Reply:
should I take this out then?

Reply:
Im afraid that I as the user don't know how to get these values still, which is a problem from a user standpoint, but maybe I'm just missing something from the SFDX information given.

Reply:
@jofsky let's talk about this when we talk about MetaDeploy -- you're going to put some time on my calendar for that, right?